build.sbt

```diff
@@ -11,6 +11,8 @@ ThisBuild / developers := List(
   tlGitHubDev("djspiewak", "Daniel Spiewak")
 )
 
+ThisBuild / githubWorkflowJavaVersions += JavaSpec.temurin("17")
```

Comment:
Hmm, you've specified the 17 Java version, but the 8 version is still in GA workflow. Why is so?

Reply:
It's a `+=` so it tests on both 8 (default) and 17. I don't want to stop testing on 8.

Reply:
ah, I see 👍🏻